packages/genesys-spark-components/src/components/beta/gux-cta-group/gux-cta-group.tsx

```diff
@@ -0,0 +1,34 @@
+import { Component, h, JSX, Prop, Element } from '@stencil/core';
+
+import { GuxCTAGroupAlignment } from './gux-cta-group.types';
+
+/**
+ * As the name suggests CTA group component is used for displaying 2-3 CTAs at the end of forms or dialog-type components
+ * @prop {GuxCTAGroupAlignment} align - Defines the aligment of the buttons within the component, left or right
+ */
+@Component({
+  styleUrl: 'gux-cta-group.scss',
+  tag: 'gux-cta-group',
+  shadow: true
+})
+export class GuxCTAGroup {
+  /*
+   * Reference to the host element.
+   */
+  @Element()
+  root: HTMLElement;
+
+  /**
+   * Buttons alignment
+   */
+  @Prop()
+  align: GuxCTAGroupAlignment = 'left';
```

Comment:
Use `start` and `end` not `left` and `right` in preparation for rtl support.

Reply:
Just commenting here to flag this again

Reply:
Changed to `start` and `end` :)